admin/organizations.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Organizations
+description: Learn about managing organizations.
+---
+
+Organizations are groups that tie together users, workspaces, and images. You
+must assign all of your images and workspaces to a specific organization. An
+end-user can only access images that are assigned to the same organization they
+are.
+
+> Be sure to familiarize yourself with the [types of
+> roles](access-control/organizations.md) you can assign users within an
+> organization.
+
+## The default organization
+
+When you first set up Coder, you'll generate the default organization. You can
```

Comment:
Maybe

> A default organization will automatically be created for you.

Or something along those lines, since it doesn't require a user to invoke an action to cause the generation